app/src/main/java/com/mmm/parq/activities/DriverActivity.java

```diff
@@ -223,13 +215,16 @@ public boolean onNavigationItemSelected(MenuItem menuItem) {
                     case R.id.drawer_history:
                         mFragment = new DriverHistoryFragment();
                         break;
-                    case R.id.drawer_settings:
-                        //mFragment = new DriverSettingsFragment();
-                        logOut();
-                        break;
+                    // case R.id.drawer_settings:
+                    //    mFragment = new DriverSettingsFragment();
+                    //    break;
```

Comment:
are all future settings going to be in the account profile? we probably won't have other settings realistically